dev/CommonStyles/CalendarDatePicker_themeresources.xaml

```diff
@@ -14,40 +14,55 @@
             <StaticResource x:Key="CalendarDatePickerForeground" ResourceKey="TextFillColorPrimaryBrush" />
             <StaticResource x:Key="CalendarDatePickerForegroundDisabled" ResourceKey="TextFillColorDisabledBrush" />
             <StaticResource x:Key="CalendarDatePickerCalendarGlyphForeground" ResourceKey="TextFillColorSecondaryBrush" />
+            <StaticResource x:Key="CalendarDatePickerCalendarGlyphForegroundPointerOver" ResourceKey="TextFillColorSecondaryBrush" />
```

Comment:
Looks like Picker updates are unrelated. Can you undo the changes not related to content dialog ?

Reply:
my bad, updated. thanks!